bazel/bazel.bzl

```diff
@@ -1,9 +1,20 @@
 # Wrappers around native build recipes to enforce consistent use of flags and build variables.
 
+load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library", "cc_test")
+
 _DEFAULT_COPTS = ["-Wall", "-Wextra"]
 
-def xx_library(name, deps = [], srcs = [], hdrs = [], copts = [], defines = [], includes = [], textual_hdrs = []):
-    native.cc_library(name = name, deps = deps, srcs = srcs, hdrs = hdrs, copts = _DEFAULT_COPTS + copts, defines = defines, includes = includes, textual_hdrs = textual_hdrs)
+def authsvc_cc_library(name, deps = [], srcs = [], hdrs = [], copts = [], defines = [], includes = [], textual_hdrs = []):
+    cc_library(name = name, deps = deps, srcs = srcs, hdrs = hdrs, copts = _DEFAULT_COPTS + copts, defines = defines, includes = includes, textual_hdrs = textual_hdrs)
+
+def authsvc_cc_binary(name, deps = [], srcs = [], copts = [], defines = []):
+    cc_binary(name = name, deps = deps, srcs = srcs, copts = _DEFAULT_COPTS + copts, defines = defines)
 
-def xx_binary(name, deps = [], srcs = [], copts = [], defines = []):
-    native.cc_binary(name = name, deps = deps, srcs = srcs, copts = _DEFAULT_COPTS + copts, defines = defines)
+def authsvc_cc_test(name, deps = [], srcs = [], data = []):
+    cc_test(
+        name = name,
+        deps = deps,
+        srcs = srcs,
+        data = data,
+        linkstatic = True
```

Comment:
I ran into a problem due to the dynamic library symbol resolution. This is a workaround strategy to resolve that. It doesn't have any effect.

Reply:
So dyanmic lib resolution would fail for the test to depend on fips boringssl? Do you mind pasting the error message?
Can you explain why `linkstatic=true` is a workaround? In another words, do we need better solution later?

Reply:
Yes. The logs posted here. https://gist.github.com/Shikugawa/0ff7ef056cf6fdb2605ad81fcb0be814
I expect more clear linkopt for dynamic resolution. But I couldn't find a clear way of it. And we don't have to stick to be dynamic link here. This is why I changed to use `linkstatic=True`.

Reply:
Got it. Could you make a comment above `linkstatic=true` with the reason why it's used? something like `We choose to use static link because boringssl FIPS build seem not be able to resolved for unit test, <gist-link> (optional)`.